google/cloud/aiplatform/datasets.py

```diff
@@ -96,17 +94,9 @@ def create(
                 Required. The user-defined name of the Dataset.
                 The name can be up to 128 characters long and can be consist
                 of any UTF-8 characters.
-            metadata_schema_uri (str):
```

Comment:
Replace `metadata_schema`, `source`, `source (GCS)`, `bq_source` and `data_items_labels` with `source (SourceConfig interface)`